routing/pathfind.go

```diff
@@ -538,20 +538,33 @@ func getOutgoingBalance(node route.Vertex, outgoingChans map[uint64]struct{},
 			}
 		}
 
-		bandwidth, ok := bandwidthHints.availableChanBandwidth(
+		bandwidth, err := bandwidthHints.availableChanBandwidth(
 			chanID, 0,
 		)
-
-		// If the bandwidth is not available, use the channel capacity.
-		// This can happen when a channel is added to the graph after
-		// we've already queried the bandwidth hints.
-		if !ok {
-			bandwidth = lnwire.NewMSatFromSatoshis(channel.Capacity)
-
-			log.Warnf("ShortChannelID=%v: not found in the local "+
-				"channels map of the bandwidth manager, "+
-				"using channel capacity=%v as bandwidth for "+
-				"this channel", shortID, bandwidth)
+		if err != nil {
+			// If the channel is not in our local channels map, use
+			// the channel capacity as a fallback. This can happen
```

Comment:
Yep this is accurate, as this is a per session snapshot. We could try to make this update dynanmically with newly mined channels, but I don't think that's really worth the extra effort.